src/java.base/share/classes/sun/net/www/http/KeepAliveCache.java

```diff
@@ -392,10 +416,11 @@ public boolean equals(Object obj) {
         if ((obj instanceof KeepAliveKey) == false)
             return false;
         KeepAliveKey kae = (KeepAliveKey)obj;
+        // SapMachine 2024-04-12: Provide additional key field for KeepAliveCache entries
```

Comment:
Maybe move comment down to the modified code?

Reply:
Done.